integration-cli/docker_cli_logs_test.go

```diff
@@ -72,21 +69,16 @@ func (s *DockerSuite) TestLogsTimestamps(c *check.C) {
 
 	lines := strings.Split(out, "\n")
 
-	if len(lines) != testLen+1 {
-		c.Fatalf("Expected log %d lines, received %d\n", testLen+1, len(lines))
-	}
+	c.Assert(lines, checker.HasLen, testLen+1)
 
 	ts := regexp.MustCompile(`^.* `)
 
 	for _, l := range lines {
 		if l != "" {
 			_, err := time.Parse(timeutils.RFC3339NanoFixed+" ", ts.FindString(l))
-			if err != nil {
-				c.Fatalf("Failed to parse timestamp from %v: %v", l, err)
-			}
-			if l[29] != 'Z' { // ensure we have padded 0's
-				c.Fatalf("Timestamp isn't padded properly: %s", l)
-			}
+			c.Assert(err, checker.IsNil, check.Commentf("Failed to parse timestamp from %v", l))
+			// ensure we have padded 0's
+			c.Assert(l[29] == 'Z', checker.Equals, true, check.Commentf("Timestamp isn't padded properly: %s", l))
```

Comment:
Apparently it doesn't work. https://github.com/docker/docker/pull/16969#discussion_r41909549